e2e-tests/portal/cypress/e2e/journeys/admin/restrict-access/restrict-access.spec.js

```diff
@@ -1,40 +1,53 @@
 const relative = require('../../../relativeURL');
-const MOCK_USERS = require('../../../../fixtures/users');
-
-const validUsers = ['ADMIN', 'UKEF_OPERATIONS'];
-const invalidUsers = ['BANK1_MAKER1', 'BANK1_CHECKER1', 'EDITOR'];
+const {
+  ADMIN,
+  UKEF_OPERATIONS,
+  BANK1_MAKER1,
+  BANK1_CHECKER1,
+  EDITOR,
+  BANK1_READ_ONLY,
+} = require('../../../../fixtures/users');
 
 context('Only allow authorised users to access admin pages', () => {
-  it('should allow Admins access to restricted pages', () => {
-    const user = MOCK_USERS[validUsers[0]];
-    cy.login(user);
-    cy.visit('/admin/users/');
-    cy.url().should('eq', relative('/admin/users/'));
-  });
+  const authorisedRoles = [{
+    roleName: 'Admins',
+    userWithRole: ADMIN,
+  }, {
+    roleName: 'UKEF Operations',
```

Comment:
This role will be removed.

Reply:
Yep - we have a separate sub-task for this :)